scenarious/scenario.py

```diff
@@ -17,10 +17,10 @@ class ScenariousException(BaseError):
 class Scenario(object):
 
     @classmethod
-    def load(cls, source, type_handlers, load_priority=None, reference_handler=None, entity_store=None, autobuild=True):
+    def load(cls, sources, type_handlers, load_priority=None, reference_handler=None, entity_store=None, autobuild=True):
         """
         Builds the Scenario based on the scenario definition stored in source, using the provided type handlers
-        :param source: A config file path or config file object to load the scenario from or a dict already built
+        :param sources: A list of config file path or config file object to load the scenario from or a dict already built
```

Comment:
wait, this make mandatory to pass a list? could be better to allow one or a list of many.

Reply:
oh, I see the if below, so, just try to clarify it in the docstring.